kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -327,7 +327,7 @@ netIPv4ToolTipStatusUnmanaged=Interface is not managed by Kura
 netIPv4ToolTipStatusL2Only=Only Layer 2 is configured
 netIPv4ToolTipStatusEnabledLAN=Interface is used to create a local network.  Use this for creating an access point that other devices can connect to.
 netIPv4ToolTipStatusEnabledWAN=Interface is enabled for use as a gateway to an external network. Use this to connect to a router, wireless access point, or cellular network.
-netIPv4ToolTipPriority=Priority used to determine which interface select as primary WAN.<br><br>In case the primary WAN interface loses connection, then the next highest priority interface is selected.<br><br>Lower values correspond to higher priority.<br><br>A value of -1 means that the metric is chosen automatically based on the device type.
+netIPv4ToolTipPriority=Priority used to determine which interface select as primary WAN.<br><br>In case the primary WAN interface loses connection, then the next highest priority interface is selected.<br><br>Lower values correspond to higher priority.<br><br>If this field is set to an empty value or to -1, it means that the metric is chosen automatically based on the device type.
```

Comment:
This should be the translation fot the whole `netIpv4ToolTipPriority` property:

```
どのインターフェイスをプライマリ WAN として選択するかを決定するために使用される優先順位。 プライマリ WAN インターフェイスが接続を失った場合、次に優先度の高いインターフェイスが選択されます。 値が小さいほど優先度が高くなります。 このフィールドが空の値または -1 に設定されている場合、メトリックがデバイス タイプに基づいて自動的に選択されることを意味します。
```

I've asked a Japanese colleague to review it, as soon as he replies I will ping you back.